### ISSUE TITLE
Twilio SMS - Mass SMS Integration (Direct)

### ISSUE THREAD

**Issue by JovanS-BAD** [erxes/erxes#2017]
**Is your feature request related to a problem? Please describe.**
Currently pricing to pay sanchat or whatever the name is to get integration for Twilio only is too much 499$ a month, that money can be well spent to sending sms.

I would rather suggest to have Direct API communication to Twilio and to integrate their services.
SMS as fast as you can, others too Voice call. So people can call directly from your app cell phones via Twilio IPO.

**Describe the solution you'd like**
So basically in Integrations instead of using other 3rd party service, we would have Twilio API where we enter API key, SID number and so on.. It would be directly communicate to Twilio without 3rd party.

**Comment by JovanS-BAD** [comment]
Close this issue. As other API service will be developed by end of june for Mass SMS.
